WordPress/Classes/Utility/Blogging Reminders/PromptRemindersScheduler.swift

```diff
@@ -0,0 +1,187 @@
+import Foundation
+import UserNotifications
+
+/// Encapsulates the local notification scheduling logic for Blogging Prompts.
+///
+class PromptRemindersScheduler {
+    enum Errors: Error {
+        case invalidSite
+        case unknown
+    }
+
+    private static var gmtTimeZone = TimeZone(secondsFromGMT: 0)!
```

Comment:
Can this not be force unwrapped please?

Reply:
Done in https://github.com/wordpress-mobile/WordPress-iOS/pull/18693/commits/452b045fa1925fee0d8754aa05dc13c2e33648b8. Thanks!